src/read.jl

```diff
@@ -192,6 +209,13 @@ function read_demand_flexibility(filepath, interval)::DemandFlexibility
             )
             demand_flexibility["duration"] = interval
         end
+
+        # Prevent the rolling_balance constraint according to the duration parameter
+        if demand_flexibility["rolling_balance"]
+            if demand_flexibility["duration"] == interval
```

Comment:
Just to confirm, I remember we project any user specified duration that is greater than the interval to the length of the interval, right?

Reply:
Yes, see https://github.com/Breakthrough-Energy/REISE.jl/blob/lane/separate_demand_flex/src/read.jl#L203

Reply:
Correct (I believe the code to which you are referring is listed directly above this snippet). I added this check because if the size of the rolling window is equal to the interval (at this point, the size of the rolling window cannot be larger than the interval due to the code about which you originally asked), then the `interval_balance` constraint and the `rolling_balance` constraint would be redundant.